### ISSUE TITLE
[suggestion] Make Freedom GPT executable on Colab

### ISSUE THREAD

**Issue by Juan-Cruz-Iturrioz** [ohmplatform/freedom-gpt-electron-app#14]
I would like to suggest an improvement to Freedom GPT. It would be great if the model could be executed on Colab, This would allow for easy access and execution of the model on a cloud-based environment, without the need for local installations.

Please consider this suggestion and let me know if there are any plans to implement it in the future. Thank you for your hard work on Freedom GPT!

**Comment by cooperdk** [comment]
Why... why? Also, eh, WHY.

**Comment by AntonioCiolino** [comment]
Faster CPU/GPU for trying it out before buying a decent ML machine?

**Comment by Nguyenpham89** [comment]
Ok

Tải Outlook for iOS<https://aka.ms/o0ukef>
________________________________
Từ: AntonioCiolino ***@***.***>
Đã gửi: Friday, April 14, 2023 7:36:10 PM
Đến: ohmplatform/FreedomGPT ***@***.***>
Cc: Subscribed ***@***.***>
Chủ đề: Re: [ohmplatform/FreedomGPT] [suggestion] Make Freedom GPT executable on Colab (Issue #14)


Faster CPU/GPU for trying it out before buying a decent ML machine?

**Comment by Nguyenpham89** [comment]
Vâng tôi nâng cấp full thử. Tôi làm trên  điện thoại nên cấu hình yếu .mọi người hãy làm thao ý mình nha.sorry

**Comment by Nguyenpham89** [comment]
Tôi cũng mới học nên chưa hiểu nhiều , tôi chỉ tạo ra những ứng dụng thôi .

**Comment by cooperdk** [comment]
I am pretty sure you need to write in English here.

**Comment by cooperdk** [comment]
> Faster CPU/GPU for trying it out before buying a decent ML machine?

Yeah, no. The binary behind (alpaca) does not support GPU at all so your own computer would be faster unless you use a (very) old laptop or have very little memory (this will require af little as 4-6 GB to run apart from what you OS uses).

**Comment by AntonioCiolino** [comment]
Heh- I have a pair of older machines that run the app but I don’t get responses back after ten minutes. One windows one Mac. dual core 2.7ghz 16mb ram.

**Comment by Nguyenpham89** [comment]
Là vì mình xử lý mọi việc qua diện thoại nên mình dùng 4-6GB còn nếu bạn sủ dụng macbook hay win 11 và lux thì cấu hình phải tăng lên . Để phù hợp hơn . Cũng đừng chỉnh cấu hình cao quá , ở mức vừa là đủ . Chúc bạn thành công

Tải Outlook for iOS<https://aka.ms/o0ukef>
________________________________
Từ: AntonioCiolino ***@***.***>
Đã gửi: Saturday, April 15, 2023 10:33:05 PM
Đến: ohmplatform/FreedomGPT ***@***.***>
Cc: Nguyên ***@***.***>; Comment ***@***.***>
Chủ đề: Re: [ohmplatform/FreedomGPT] [suggestion] Make Freedom GPT executable on Colab (Issue #14)


Heh- I have a pair of older machines that run the app but I don’t get responses back after ten minutes. One windows one Mac. dual core 2.7ghz 16mb ram.

**Comment by ItsPi3141** [comment]
why? if you want to use colab, then just use oobabooga text generation webbui or the llama.cpp CLI.
